src/App.jsx

```diff
@@ -0,0 +1,23 @@
+import './App.css';
+import Home from './views/Home/Home';
+import { Route, BrowserRouter as Router, Switch } from 'react-router-dom';
+import Movie from './views/Movie/Movie';
+
+function App() {
+  return (
+    <div className="App">
+      <Router>
+        <Switch>
+          <Route exact path="/">
+            <Home />
+          </Route>
+          <Route exact path="/:movieId">
```

Comment:
Recommend avoiding using a :id with the root path. 
Something like /movie/:id is better.